packages/cdk8s-plus-17/src/container.ts

```diff
@@ -248,9 +257,9 @@ export interface ContainerProps {
 export class Container {
 
   /**
-   * The port this container exposes.
+   * List of ports this container exposes.
    */
-  public readonly port?: number;
+  public readonly ports: ContainerPort[] = [];
```

Comment:
This change would be the breaking change. Should I create ```public get port() { return this.ports[0]?.port; }``` not to bring the breaking change?